scalalib/overrides/scala/math/ScalaNumericConversions.scala

```diff
@@ -0,0 +1,119 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___     Scala API                            **
+**    / __/ __// _ | / /  / _ |    (c) 2007-2013, LAMP/EPFL             **
+**  __\ \/ /__/ __ |/ /__/ __ |    http://scala-lang.org/               **
+** /____/\___/_/ |_/____/_/ | |                                         **
+**                          |/                                          **
+\*                                                                      */
+
+package scala
+package math
```

Comment:
If you need to introduce an `overrides` file, first have one commit where you copy the file from the Scala sources verbatim, with the following commit message: "Add scala.math.ScalaNumericConversions" in overrides to track changes." Then, in the main commit where you implement big numbers, change what needs to be changed.
This way, the git history clearly shows what's different between upstream and our "fork".